openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1392,18 +1372,17 @@ def get_partial_charges(self, molecule: "Molecule", **kwargs) -> "unit.Quantity"
                 f"{type(molecule)}"
             )
 
-        _, top_with_charges = self.create_openmm_system(
-            molecule.to_topology(), return_topology=True, **kwargs
+        # Bad type annotation in Interchange, expect to be fixed in 0.3.1
```

Comment:
See https://github.com/openforcefield/openff-interchange/pull/653/files#diff-bcc2b52dee6f379dfc70db4239da71316c5cd2ec9fabc4e8e5c81958517530f9L264-R266, which for some reason was not picked up there.